src/main/resources/static/styles/custom_bootstrap.css

```diff
@@ -29,6 +30,24 @@
     --bs-gradient: none;
 }
 
+.btn-primary {
```

Comment:
may be simplified

Reply:
removed the settings for disabled because it will never be disabled